app/models/chargeback.rb

```diff
@@ -59,6 +59,10 @@ def self.build_results_for_report_chargeback(options)
         metric_rollup_record = metric_rollup_records.first
         rates_to_apply = rates.get(metric_rollup_record)
 
+        if metric_rollup_record.resource_type == Container.name && rates_to_apply.empty?
+          rates_to_apply = [ChargebackRate.find_by_description("Default Container Image Rate")]
+        end
+
```

Comment:
`rates_to_apply.empty?` is it enough ? can be returned here also storage rates ? (maybe by definition, just asking)